File: java/CTimer.java
package cell;

public class CTimer 
{
	long m_begin=System.currentTimeMillis();
	public int Delay()
	{
		long before=m_begin;
		m_begin=System.currentTimeMillis();
		return (int) (System.currentTimeMillis()-before);
	}

}
class CFrame
{
	static int m_addTime;
	static CTimer m_time=new CTimer();
	static int m_count;
	static int m_lastFrame;

	public static int Get()
	{
		m_addTime += m_time.Delay();
		if (m_addTime > 1000)
		{
			m_addTime -= 1000;
			m_lastFrame = m_count;
			m_count = 0;
		}
		m_count++;

		return m_lastFrame;
	}
};

File: js/CTimer.js
function CTimer() 
{
	this.m_begin=0;
	this.Delay();
}
CTimer.prototype.constructor	=CTimer;
CTimer.prototype.Delay=function()
{
	if(this.m_begin==0)
	{
		this.m_begin = new Date().getTime();
		return 0;
	}
	var before = this.m_begin;
	this.m_begin = new Date().getTime();
	return this.m_begin - before;
}

function DfFrame() 
{
	this.m_addTime=0;
	this.m_time=new CTimer();
	this.m_count = 0;
	this.m_lastFrame = 0;
}
DfFrame.prototype.constructor	=DfFrame;

DfFrame.prototype.Get=function()
{
	return this.m_lastFrame;
}
DfFrame.prototype.Update=function()
{
	this.m_addTime += this.m_time.Delay();
	if (this.m_addTime > 1000)
	{
		this.m_addTime -= 1000;
		this.m_lastFrame = this.m_count;
		this.m_count = 0;
	}
	this.m_count++;
}
var CFrame=new DfFrame();